src/plugins/GUITestBase/src/harness/UGUITest.cpp

```diff
@@ -88,6 +89,10 @@ static QString getDataDirImpl() {
     } else {
         dataDir = QString("data/");
     }
+
+    if (!QFileInfo(dataDir).exists()) {
+        dataDir = BundleInfo::getDataSearchPath() + "/";
```

Comment:
getDataSearchPath may return empty string and in this case you will have a root "/" dir as dataDir which is not correct.

Reply:
yes, fixed